olive/model.py

```diff
@@ -362,13 +415,49 @@ def load_model(self, rank: int = None) -> torch.nn.Module:
     def prepare_session(self, inference_settings: Dict[str, Any], device: Device, rank: int = None):
         return self.load_model().eval()
 
+    def get_dummy_inputs(self):
```

Comment:
these logic seems not be related to PyTorch model. Instead, it should be part of dataset.

Reply:
It is part of pytorch model. When converting to onnx or during qat, we need dummy input.

Reply:
Can you please unblock this PR? This is urgent for whisper model support.

Reply:
Added TODO as discussed offline.